cpp/oneapi/dal/algo/knn/backend/model_conversion.hpp

```diff
@@ -59,10 +59,9 @@ inline auto convert_onedal_to_daal_knn_model(const model<Task>& m) {
 
     // Changed to perform a copy as far as we have similar logic
     // for d4p patching; to allign performance with DAAL
```

Comment:
please remove this comment, due to now you aligned the behavior with sycl version
https://github.com/uxlfoundation/oneDAL/pull/3310/files#diff-1faadc975bf80f6f4baa5b1e7c09984cbe4a2c19f2ac7a3ddfa8c6a434049e22R74